src/coreclr/jit/gentree.h

```diff
@@ -1114,6 +1114,7 @@ struct GenTree
     inline bool IsRegOptional() const;
     inline void SetRegOptional();
     inline void ClearRegOptional();
+    inline void ClearContainedAndRegOptional();
```

Comment:
Note `ClearContained` already clears `RegOptional`, so should be no need for this function.

Reply:
Ah, gotcha didn't know that was the case. thanks